app/code/community/Drip/Connect/etc/system.xml

```diff
@@ -109,8 +109,8 @@
                             <frontend_type>text</frontend_type>
                             <sort_order>60</sort_order>
                             <show_in_default>1</show_in_default>
-                            <show_in_website>1</show_in_website>
-                            <show_in_store>1</show_in_store>
+                            <show_in_website>0</show_in_website>
+                            <show_in_store>0</show_in_store>
```

Comment:
We only ever used memory limit in global scope, so stop allowing it to be set in local scope.

On an unrelated note, manually setting the memory limit like we do is incredibly ugly, and we should stop doing that.